docs/tasks/semester-checklist.rst

```diff
@@ -1,9 +1,39 @@
 Semester Checklist
 ==================
-
 A quick checklist of tasks that need to be completed at the start of every semester.
 
--  Reserve the meeting room by speaking to the lab manager (requires a signature from the president and vice president). Get set up with swipe access if need be.
+Before Semester
```

Comment:
I still feel it would be better to have *Before Fall Semester* and *Before Spring semester* instead of this here. The current layout feels a little confusing, and would make better sense if you have only *Before Semester* or both semesters listed.

Reply:
**Before Fall Semester**
- Hold weekly/bi-weekly eboard meetings
- Create *Semester Schedule Planning* issue
- Reserve weekly club meeting space
- Triage and review tasks from previous semester to plan for upcoming semester (see `project board <https://github.com/orgs/RITlug/projects/1?fullscreen=true>`__).

**Before Spring Semester**
- Hold meeting(s) to discuss plans for next semester
- Review tasks from previous semester

------------

I feel something along the lines of this would make the flow of the document go a little smoother. What do you think?

Reply:
Don't we need to reserve the meeting space before the start of both semester @Tjzabel? Last year we switched rooms between semesters, and this year we currently only have the room reserved for the fall semester. 

I initially didn't add a spring semester list because there was nothing unique about that semester  which we didn't have in the fall semester prep as far as I know. I can add the spring semester separately, but it will contain 4 of the same items as the fall semester.

I will try to think of a way to re-word this to make it more clear and avoid duplicate information.

Reply:
@jrtechs it seems I had a pretty large brain fart. Yes, I do believe we reserve meeting spaces for each semester, not yearly. 
In that case, you are totally fine with having just a *Before Semester* section. There are 2-3 items that are reserved for Fall Semester planning such as financial certs and club gala, so just make sure to mention them as such.

In that case, it might be good to restructure as followed:

**Before Semester**
- Get all eboard members financially certified through Campus Life. (***Fall Semester Only***)
- Plan for Activities Gala (***Fall Semester Only***)
- Hold weekly/bi-weekly eboard meetings.
- Discuss plans and brainstorm for next semester.
- Reserve meeting space.
- Triage and review tasks from previous semester to plan for coming semester (see `project board <https://github.com/orgs/RITlug/projects/1?fullscreen=true>`__).

**At Start of Semester**
- (...)